kie-dmn/kie-dmn-core/src/main/java/org/kie/dmn/core/assembler/DMNAssemblerService.java

```diff
@@ -93,12 +95,13 @@ public void addResource(Object kbuilder, Resource resource, ResourceType type, R
     }
 
     private List<DMNProfile> getDMNProfiles(KnowledgeBuilderImpl kbuilderImpl) {
```

Comment:
Assembler code modified to be the unique source of entrance for getting "default profiles" and the likes, as discussed.